### PR TITLE
Add keypoints detection quickstart example integration

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
Follows this [quickstart](https://docs.kolena.io/quickstart/quickstart-building-a-workflow) with minor tweaks.

### CODE REVIEW

**Comment by gordonhart** [comment]
Should this be `keypoints` (plural)?

Edit: discussed internally, going with `Keypoint Detection`

File: examples/keypoint_detection/tests/test_keypoints_detection.py
Comment (gordonhart) [inline examples/keypoint_detection/tests/test_keypoints_detection.py:1]: Nit: rename to `test_keypoint_detection.py`

**Comment by gordonhart** [comment]
Will need to enable CI for these tests with the following steps:

1. Add the `keypoint_detection` subproject to the [list of subprojects in the matrix parameters](https://github.com/kolenaIO/kolena/blob/trunk/.circleci/config.yml#L228)
2. Add an `exclude` block for the `large` resource class to ensure this is only run on `small`

File: examples/keypoint_detection/keypoint_detection/evaluator.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class KeypointsEvaluator(Evaluator):
+    """See API documentation for details."""
+
+    def compute_test_case_metrics(self, test_case, inferences, metrics, configuration=None):
+        # Generate dummy metrics for demo purposes.
+        return TestCaseMetrics(random(), random())
+
+    def compute_test_sample_metrics(self, test_case, inferences, configuration=None):
+        # Generate dummy metrics for demo purposes.
+        return [(i[0], TestSampleMetrics(random(), bool(random() > 0.5))) for i in inferences]
```
Comment (gordonhart) [inline examples/keypoint_detection/keypoint_detection/evaluator.py]: Do we have a real evaluator implementation for this workflow?

File: examples/keypoint_detection/keypoint_detection/evaluator.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
+from typing import Dict
```
Comment (ly-kolena) [inline examples/keypoint_detection/keypoint_detection/evaluator.py:14]: Missing copyright headers?

File: .circleci/config.yml [hunk 3]
```diff
@@ -225,11 +225,14 @@ workflows:
...
           matrix:
             parameters:
-              subproject: [ text_summarization, age_estimation ]
+              subproject: [ keypoint_detection, text_summarization, age_estimation ]
               resource-class: [ large, small ]
               python-version: [ "3.9" ]
             # TODO: not sure how to pair subproject to resource class without declaring as matrix and adding excludes
             exclude:
+              - subproject: keypoint_detection
+                resource-class: small
```
Comment (ly-kolena) [inline .circleci/config.yml]: Can you confirm this workflow needs resource large?

**Review by ly-kolena — APPROVED** [review]

**Comment by ly-kolena** [comment]
Side benefit of using non-transient database, can checkout the nice examples [here](https://app.kolena.io/sdk-integ/studio?workflow=Keypoint+Detection&testSuiteId=1024&testCaseId=1929&testSampleId=4224).

**Pull request merged**